.github/workflows/macro-tests.yml

```
@@ -35,3 +35,29 @@ jobs:
         continue-on-error: true
         run: Tools/measure-macro-performance.sh --iterations 5
+
+  path-identity:
+    runs-on: macos-26
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          path: not-innodi
+      - name: Select Xcode 26.3
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2010:warning:3:3: Don't use ls | grep. Use a glob or a for loop with a condition to allow non-alphanumeric filenames

(shellcheck)